.github/workflows/version.yml

```
@@ -15,13 +15,11 @@ jobs:
                   fetch-depth: 0
                   token: ${{ secrets.OS_BOTIFY_TOKEN }}
 
-            - name: Setup Node
-              uses: actions/setup-node@v1
+            - uses: softprops/turnstyle@v1
```

Let's use commit hash here since it has `GITHUB_TOKEN`

```suggestion
            - uses: softprops/turnstyle@8db075d65b19bf94e6e8687b504db69938dc3c65
```